doc/examples/jupyter_notebook/Makefile

```diff
@@ -1,12 +1,11 @@
 build:
 	docker build -t pachyderm_jupyter .
 
-prep: build
+prep:
 	pachctl version
-	./citibike_data/load.sh
-	./weather_data/load.sh
+	cd citibike_data && ./load.sh
+	cd weather_data && ./load.sh
 	pachctl create-pipeline -f ./sales_data/pipeline.json
-	pachctl create-job -f jupyter.json
```

Comment:
We can.  I will work on that.  I separated it out for now to at least force the separation of the two commands.